app/models/trip.rb

```diff
@@ -0,0 +1,8 @@
+class Trip < ApplicationRecord
```

Comment:
Should add a comment explaining all of our thinking behind trips and the assumptions we make, such as that all trips require a parent permission slip.